src/main/java/step3/ui/FastScanner.java

```diff
@@ -0,0 +1,56 @@
+package step3.ui;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.StringTokenizer;
+
+public class FastScanner {
+    private BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+    private StringTokenizer st = new StringTokenizer("");
+
+    private static FastScanner fs = new FastScanner();
+
+    private FastScanner() {
+    }
+
+    public static FastScanner of() {
+        return fs;
+    }
+
+    public String next() {
+        while (!st.hasMoreTokens())
+            try {
+                st = new StringTokenizer(br.readLine());
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
```

Comment:
```suggestion
        while (!st.hasMoreTokens()) {
            try {
                st = new StringTokenizer(br.readLine());
            } catch (IOException e) {
                e.printStackTrace();
            }
        }
```
블록은 항상 추가해주세요! 대부분의 코딩 컨벤션에서 추가하는 것을 권장한답니다.